scripts/convert_csv_to_json.py

```diff
@@ -10,43 +10,47 @@ def generate_json_files():
     source_root = './source-data'
     output_root = './data'
 
+    for source_region_directory_name in os.listdir(source_root):
```

Comment:
This can be simplified significantly by using `os.walk` - it is a generator that returns a tuple of (root, directory names, file names), so can be unpacked where required.

Reply:
I'd used `walk` in the first iteration of this file and a discussion was had which resulted in me changing to this pattern. It was essentially due to us knowing (and wanting to limit) the source and output folder structures, and not having to deal with random/arbitrary levels of nesting.

Reply:
Ok that makes sense 👍